Error: Cannot find module 'chalk'

Hi, today I did a serverless deployment after a couple of weeks from the last one but I'm getting an error related to Chalk.

As I'm still using serverless V3, should I stick to the plugin's 1.2.0 version? Thanks in advance!

```
sls deploy --verbose --stage staging
Environment: linux, node 20.18.1, framework 3.40.0, plugin 7.2.3, SDK 4.5.1
Docs:        docs.serverless.com
Support:     forum.serverless.com
Bugs:        github.com/serverless/serverless/issues
Error:
Error: Cannot find module 'chalk'
Require stack:
- /usr/lib/node_modules/serverless-exports-plugin/dist/index.js
- /usr/lib/node_modules/serverless/lib/utils/require-with-import-fallback.js
- /usr/lib/node_modules/serverless/lib/classes/plugin-manager.js
- /usr/lib/node_modules/serverless/lib/serverless.js
- /usr/lib/node_modules/serverless/scripts/serverless.js
- /usr/lib/node_modules/serverless/bin/serverless.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1225:15)
    at Module._load (node:internal/modules/cjs/loader:1051:27)
    at Module.require (node:internal/modules/cjs/loader:1311:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (/usr/lib/node_modules/serverless-exports-plugin/dist/index.js:7:33)
    at Module._compile (node:internal/modules/cjs/loader:1469:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1548:10)
    at Module.load (node:internal/modules/cjs/loader:1288:32)
    at Module._load (node:internal/modules/cjs/loader:1104:12)
    at Module.require (node:internal/modules/cjs/loader:1311:19)
    at require (node:internal/modules/helpers:179:18)
    at module.exports (/usr/lib/node_modules/serverless/lib/utils/require-with-import-fallback.js:5:17)
    at PluginManager.requireServicePlugin (/usr/lib/node_modules/serverless/lib/classes/plugin-manager.js:182:12)
    at PluginManager.resolveServicePlugins (/usr/lib/node_modules/serverless/lib/classes/plugin-manager.js:198:29)
    at PluginManager.loadAllPlugins (/usr/lib/node_modules/serverless/lib/classes/plugin-manager.js:136:47)
    at Serverless.init (/usr/lib/node_modules/serverless/lib/serverless.js:146:30)
    at async /usr/lib/node_modules/serverless/scripts/serverless.js:601:7
make: *** [Makefile:38: deploy-staging] Error 1
```

--- a/package.json
+++ b/package.json
@@ -39,7 +39,6 @@
     "@types/node": "^20.5.0",
     "@types/serverless": "^3.12.14",
     "aws-sdk": "^2.1437.0",
-    "chalk": "^4.0.0",
     "execa": "^7.2.0",
     "js-yaml": "^4.1.0",
     "json-schema-to-ts": "^2.9.2",
@@ -49,6 +48,9 @@
     "serverless-localstack": "^1.1.1",
     "typescript": "^5.1.6",
     "vitest": "^0.34.1"
+  },
+  "dependencies": {
+    "chalk": "^4.0.0"
   },
   "peerDependencies": {
     "serverless": "^3.0.0 || ^4.0.0"

